UKHO.ExternalNotificationService.API/UKHO.ExternalNotificationService.API/appsettings.json

```diff
@@ -1,10 +1,31 @@
 {
+    "EventHubLoggingConfiguration": {
+        "MinimumLoggingLevel": "Warning",
+        "UkhoMinimumLoggingLevel": "Information",
+        "Environment": "dev",
+        "EntityPath": "",
+        "System": "ENS API",
+        "Service": "External Notification Service",
+        "NodeName": "Azure",
+        "ConnectionString": ""
+    },
   "Logging": {
     "LogLevel": {
       "Default": "Information",
-      "Microsoft": "Warning",
+      "Microsoft.AspNetCore": "Warning",
       "Microsoft.Hosting.Lifetime": "Information"
+    },
+        "ApplicationInsights": {
+            "LogLevel": {
+                "Default": "Information"
     }
   },
+        "PathFormat": "Logs/UKHO.External Notification Service.API-{Date}.txt",
+        "Json": true
+    },
```

Comment:
"PathFormat": "Logs/UKHO.External Notification Service.API-{Date}.txt",
   "Json": true
Please remove these two properties from appsettings as these are not default properties for Logging.

Reply:
Yes. Removed from appsetting.json file.
Thanks!